cookbook/bundles/best_practices.rst

```diff
@@ -72,48 +68,43 @@ class name.
     :class:`Symfony\\Bundle\\FrameworkBundle\\FrameworkBundle`.
 
 Each bundle has an alias, which is the lower-cased short version of the bundle
-name using underscores (``acme_hello`` for ``AcmeHelloBundle``, or
-``acme_social_blog`` for ``Acme\Social\BlogBundle`` for instance). This alias
-is used to enforce uniqueness within a bundle (see below for some usage
-examples).
+name using underscores (``acme_hello`` for ``AcmeHelloBundle``). This alias
+is used to enforce uniqueness within a bundle (see below for some usage examples).
 
 Directory Structure
 -------------------
 
-The basic directory structure of a HelloBundle must read as follows:
+The basic directory structure of a AcmeHelloBundle must read as follows:
 
 .. code-block:: text
 
-    XXX/...
-        HelloBundle/
-            HelloBundle.php
-            Controller/
-            Resources/
-                meta/
-                    LICENSE
-                config/
-                doc/
-                    index.rst
-                translations/
-                views/
-                public/
-            Tests/
-
-The ``XXX`` directory(ies) reflects the namespace structure of the bundle.
-
-The following files are mandatory:
-
-* ``HelloBundle.php``;
+    src/
+        ...
+        Acme/
+            HelloBundle/
+                AcmeHelloBundle.php
+                Controller/
+                Resources/
+                    meta/
+                        LICENSE
+                    config/
+                    doc/
+                        index.rst
+                    translations/
+                    views/
+                    public/
+                Tests/
+
+**The following files are mandatory** because they ensure a structure convention
+that automated tools can rely on:
+
+* ``AcmeHelloBundle.php``: This is the class that transforms a plain directory
+  into a Symfony bundle;
 * ``Resources/meta/LICENSE``: The full license for the code;
 * ``Resources/doc/index.rst``: The root file for the Bundle documentation.
 
-.. note::
-
-    These conventions ensure that automated tools can rely on this default
-    structure to work.
-
 The depth of sub-directories should be kept to the minimal for most used
-classes and files (2 levels at a maximum). More levels can be defined for
+classes and files (two levels at a maximum). More levels can be defined for
 non-strategic, less-used files.
```

Comment:
-1 on this best practice. Why should a class in a namespace `Behat\Behat\Output\Node\Printer` be less used/non-strategic as a class in `Behat\Behat\Cli`?